Support Files/es.lproj/Localizable.strings

```diff
@@ -8,7 +8,7 @@
 "preferences" = "Preferencias";
 
 // MARK: - About
-"copyright" = "Copyright © 2019 Los autores de nef";
+"copyright" = "Copyright © 2020 Los autores de nef";
```

Comment:
I would completely drop any translation, as Xcode itself is not translated. It feels weird to me when I have part of the UI in a language, and part in another one.